yarn.lock

```diff
@@ -22,7 +22,7 @@ __metadata:
   languageName: node
   linkType: hard
 
-"@babel/code-frame@npm:^7.0.0, @babel/code-frame@npm:^7.12.13, @babel/code-frame@npm:^7.22.13, @babel/code-frame@npm:^7.23.5":
```

Comment:
Actually not sure why this got in here, I was testing that my commit hook to avoid package-lock json file couldn't get committed, or past CI. must have triggered an update. Gonna back this out if i can